src/app/public/modules/media-query/media-query-listener.ts

```diff
@@ -2,4 +2,7 @@ import {
   SkyMediaBreakpoints
 } from './media-breakpoints';
 
+/**
+ * @internal
```

Comment:
This is an exported type; probably needs to be in the docs.

Reply:
Fixed, but now its throwing an error: https://github.com/blackbaud/skyux-docs-tools/issues/114

Reply:
Let's keep the `@internal` then and create an issue to re-add it after docs-tools is fixed.